packages/calling-component-bindings/review/beta/calling-component-bindings.api.md

```diff
@@ -202,6 +205,8 @@ export interface CommonCallingHandlers {
     // (undocumented)
     onStopScreenShare: () => Promise<void>;
     // (undocumented)
+    onSubmitSurvey(survey: CallSurvey, options?: SubmitSurveyOptions): Promise<CallSurveyResponse | undefined>;
```

Comment:
I guess technically because we aren't releasing a UI component this should be at the composite selector level, but idm either way